packages/react-native-lockdown/src/repair.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Repair JS intrinsics
+ *
+ * The Hermes (or vanilla) SES shim is loaded as the first Metro polyfill, so
+ * {@link repairIntrinsics} can be called.
+ *
+ * Called with lockdown options ideal for React Native apps.
```

Comment:
wanted to `@link repairOptions` here
- https://github.com/endojs/endo/pull/2845

Reply:
Cross-module linking is basically broken; unless you have a reference handy, don't worry about it